frontend/src/components/public_group/PublicGroupHero.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PublicTopBarV2 from '../../containers/PublicTopBarV2';
+
+export default class PublicGroupHero extends React.Component {
+  render() {
+    const { group } = this.props;
+    const collectiveBg = '/static/images/collectives/default-header-bg.jpg';
+
+    return (
+      <section className='PublicGroupHero relative px2 bg-black bg-cover white' style={{backgroundImage: `url(${collectiveBg})`}}>
+        <div className='container relative center'>
+          <PublicTopBarV2 className='pt3 absolute top-0 left-0 right-0' />
+          <div className='PublicGroupHero-content'>
+            <p className='PublicGroup-font-17 mt0 mb2'>Hi! This is an <span className='-fw-bold'>OpenCollective</span> by <a href={group.website} className='underline white'>{group.name}</a> and we’re on a mission to&hellip;</p>
+            <h1 className='PublicGroupHero-mission max-width-3 mx-auto mt0 mb3 white -ff-sec'>{group.description}</h1>
+            <a href='#join-us' className='mb3 -btn -btn-big -bg-green -ttu -ff-sec -fw-bold'>Be part of it!</a>
+            <p className='h6'>Scroll down to find out more.</p>
+            <svg width='14' height='9'>
+              <use xlinkHref='#svg-arrow-down' stroke='#fff'/>
```

Comment:
I think that could be a good idea (kinda expected by some users), but I would prefer to wait a bit on this, on the next PR's we will be pushing a "smooth scroll transition" for the menu items, so we can also add the scrollTo behaviour for the arrow as well.